test/HostActivationTests/GivenThatICareAboutDotnetTestXunitScenarios.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.DotNet.CoreSetup.Test;
+using System.Runtime.InteropServices;
+using System.IO;
+
+namespace Microsoft.DotNet.Tools.Publish.Tests
+{
+    public class GivenThatICareAboutDotnetTestXunitScenarios
+    {
+        private string DotnetTestXunitVersion => "1.0.0-rc2-192208-24";
+        private RepoDirectoriesProvider RepoDirectories { get; set; }
+
+        public GivenThatICareAboutDotnetTestXunitScenarios()
+        {
+            RepoDirectories = new RepoDirectoriesProvider();
+        }
+
+        [Fact]
+        public void Muxer_activation_of_dotnet_test_XUnit_on_Portable_Test_App_Succeeds()
+        {
+            var portableTestAppFixture = new TestProjectFixture("PortableTestApp", RepoDirectories)
+                .EnsureRestored(RepoDirectories.CorehostPackages, RepoDirectories.CorehostDummyPackages)
+                .BuildProject();
+
+            ActivateDotnetTestXunitOnTestProject(RepoDirectories, portableTestAppFixture);
+        }
+
+        [Fact]
+        public void Muxer_activation_of_dotnet_test_XUnit_on_Standalone_Test_App_Succeeds()
+        {
+            if ( ! RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Console.WriteLine("Skipped Test: https://github.com/dotnet/core-setup/issues/121");
```

Comment:
This has gone away. Put entrypoint true in project.json because a fix is not coming for this.

---
In reply to: [66353019](https://github.com/dotnet/core-setup/pull/97#discussion_r66353019) [](ancestors = 66353019,66345654)